clusters.go

```diff
@@ -65,3 +65,14 @@ func GetDefaultClusterFromZKUrl(zkURL string) *Cluster {
 		AgentRoot:     "/var/lib/mesos",
 	}
 }
+
+func GetDefaultMesosClusterFromZKUrl(zkURL string) *Cluster {
```

Comment:
I don't think you need to create another function for this.
instead, you can add another field in Cluster struct for `/mesos`